coderd/templateversions.go

```diff
@@ -163,6 +164,215 @@ func (api *API) templateVersionParameters(rw http.ResponseWriter, r *http.Reques
 	httpapi.Write(rw, http.StatusOK, values)
 }
 
+func (api *API) createTemplateVersionPlan(rw http.ResponseWriter, r *http.Request) {
```

Comment:
Other endpoints like this have put the method prior, so this would be `postTemplateVersionPlan`